src/Listeners/AddApiRoutes.php

```diff
@@ -38,8 +38,8 @@ public function routes(ConfigureApiRoutes $routes)
         /**
          * Forum side
          */
-        $routes->get('/masquerade/profile/{id:[0-9]+}', 'masquerade.api.profile', UserProfileController::class);
-        $routes->get('/masquerade/configure', 'masquerade.api.configure', UserConfigureController::class);
-        $routes->post('/masquerade/configure', 'masquerade.api.configure.save', UserConfigureController::class);
+        $routes->get('/u/{username}/profile/{id:[0-9]+}', 'masquerade.api.profile', UserProfileController::class);
```

Comment:
Having username and user id is redundant. Using `u/username/masquerade` might be a good idea though. Requires further change in the controller